dice-where-downloader-lib/src/main/java/technology/dice/dicewhere/downloader/destination/s3/S3DownloadSetup.java

```diff
@@ -51,7 +51,8 @@ public Pair<FileAcceptor, S3Source> setupDownload(
                           .credentialsProvider(
                               StaticCredentialsProvider.create(
                                   AwsBasicCredentials.create(c.getAwsKeyId(), c.getAwsSecretKey())))
-                          .region(Region.of(c.getAwsRegion()));
+                          .region(Region.of(c.getAwsRegion()))
+                          .forcePathStyle(true);
```

Comment:
Can we only `forcePathStyle` if we're overriding the endpoint with `endpointOverride` below? I worry about changing the behaviour for normal S3 unecessarily and resulting in worse performance or strange issues.

Reply:
sorry, thought this was test code. Agreed, we should not be changing the behaviour because of a test lib change. It should be the tests that are fixed